platform/javascript/detect.py

```diff
@@ -110,11 +110,8 @@ def configure(env):
     # No multi-threading (SharedArrayBuffer) available yet,
     # once feasible also consider memory buffer size issues.
     env.Append(CPPDEFINES=['NO_THREADS'])
-
-    # These flags help keep the file size down.
-    env.Append(CCFLAGS=['-fno-exceptions', '-fno-rtti'])
-    # Don't use dynamic_cast, necessary with no-rtti.
-    env.Append(CPPDEFINES=['NO_SAFE_CAST'])
+    # These flags help keep the file size down
+    env.Append(CPPFLAGS=["-fno-exceptions"]) #, '-fno-rtti' - dynamic_cast required by ICU
```

Comment:
I don't think we can/should do that. CC @eska014

Reply:
Yeah, turning off runtime type information saves about 200KiB last I tested, so we should avoid re-enabling it. Does ICU have some build option that avoids usage of RTTI?